dev-client/src/components/tables/DataGridTable.tsx

```diff
@@ -66,12 +66,12 @@ export const DataGridTable = ({rows, headers, ...containerProps}: Props) => {
       </Row>
       <Divider />
       {rows.map((row: (typeof rows)[number], i: number) => (
-        <>
-          <Row justifyContent="flex-start" key={i} variant="tablerow" py="10px">
+        <React.Fragment key={i}>
+          <Row justifyContent="flex-start" variant="tablerow" py="10px">
             {row.map(displayCol)}
           </Row>
           <Divider key={`divider${i}`} />
```

Comment:
Oh nice! 
Suggestion: Haven't tried it, but I imagine you could remove the key from `Divider` too